sql_generators/glean_usage/glean_app_ping_views.py

```diff
@@ -13,6 +13,7 @@
 
 from jinja2 import Environment, FileSystemLoader
 from mozilla_schema_generator.glean_ping import GleanPing
+from pathos.multiprocessing import ThreadingPool
```

Comment:
The other SQL generators that do parallel processing use `ProcessingPool` from `pathos.multiprocessing`, not `ThreadingPool`.  Is there a benefit to using `ThreadingPool`?

Also, why do the SQL generators use the third-party `pathos` package for parallel processing, while all other bigquery-etl code seems to use the built-in `multiprocessing` module?

Reply:
this was the reason we ended up using pathos: https://github.com/mozilla/bigquery-etl/pull/2567/files#r768227839

_"The standard multiprocessing module has problems with executing functions in parallel that are not part of a package (resulting in pickling errors)."_

I tried using `ProcessingPool` here, but ran into two issues: one there were some issues with pickling the `GleanAppPingViews` class due to the `self` references, and second `ProcessingPool`s cannot be nested. `glean_usage` is using `ProcessingPool` to kick off generating the different tables per app.

Reply:
Interesting.  I think that'd be worthy of an explanatory comment where `ThreadingPool` is being used in `GleanAppPingViews` (at least about why `ProcessingPool` couldn't be used).